src/views/components-default/TheDefaultWalletBanner.vue

```diff
@@ -80,9 +80,7 @@
             class="mew-body font-weight-medium footer-text"
           >
             <p>{{ $t('home.mew-wallet-modal.footer-text') }}</p>
-            <a
-              :href="isMobile() ? btn.url : 'https://mewwallet.com'"
-              target="_blank"
+            <a href="https://mewwallet.com" target="_blank"
```

Comment:
I think we still want to keep the mobile links, this link (mewwallet.com) is for desktop.

Reply:
This code is outside of the v-for loop,so the (btn) doesn't exist at this point.
I think the mobile and desktop link for this should be the same???
Please see the picture.
![1](https://user-images.githubusercontent.com/9893774/142948129-1aec8f8f-869b-4322-979e-ddeed54c10b2.png)